src/nls/root/strings.js

```diff
@@ -418,6 +418,7 @@ define({
     "CMD_SHOW_IN_EXPLORER"                : "Show in Explorer",
     "CMD_SHOW_IN_FINDER"                  : "Show in Finder",
     "CMD_SHOW_IN_OS"                      : "Show in OS",
+    "CMD_SPLITVIEW_TOGGLE"                : "Toggle Panes",
```

Comment:
As this command is for shuffling focus across panes, can we name it accordingly?